apps/perf-test/webpack.config.js

```diff
@@ -13,6 +13,9 @@ module.exports = resources.createServeConfig(
     output: {
       filename: 'perf-test.js',
     },
+    optimization: {
```

Comment:
Can you please explain why this is needed? Does module concatenation obfuscate flamegraphs?

Reply:
@kenotron and I were not totally sure here; technically I don't think this should be required. But we were seeing issues with a particular module and this was a workaround. I'll play with it a little and see if I can find any other reasons, and will also paste the error we were seeing.

Reply:
I dug in here and removed it; I believe the default library format changed and caused some things to be treeshaken incorrectly. Setting the libraryTarget to UMD as a default in webpack-resources fixed this.